.idea/codeStyleSettings.xml

```diff
@@ -1,23 +1,2 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
```

Comment:
could you please revert this change, it is code style for webstorm, i guess it was committed b y mistake